model/fieldtypes/compat/Float.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
```

Comment:
I don't think this will do anything as if the file is loaded you've already lost — with a compile-time error.